test/date_time_parser_test.exs

```diff
@@ -5,7 +5,9 @@ defmodule DateTimeParserTest do
   alias DateTimeParser
   alias DateTimeParser.Parser
 
-  doctest DateTimeParser
+  if Version.match?(System.version(), ">= 1.5.0") do
+    doctest DateTimeParser
+  end
```

Comment:
The doctests just simply don't work well with Elixir 1.4.x because of of datetimes with timezones are rendered differently, so I'm going to skip it for that version.